cw/cw305/capture_traces.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import binascii
+from Crypto.Cipher import AES
+import numpy as np
+import time
+from tqdm import tqdm
+import yaml
+
+import chipwhisperer as cw
+
+from util import device
+from util import plot
+
+
+def initialize_test(device_cfg):
+  """Initialize test setup."""
+  ot = device.OpenTitan(
+    device_cfg['fpga_bitstream'],
+    device_cfg['fw_bin'],
+    device_cfg['pll_frequency'],
+    device_cfg['baudrate'])
+  print(f'Scope setup with sampling rate {ot.scope.clock.adc_rate} S/s')
+  return ot
+
+
+def run_capture(capture_cfg, ot):
+  """Run ChipWhisperer capture.
+
+  Based on https://github.com/newaetech/chipwhisperer-jupyter/blob/master/PA_HW_CW305_1-Attacking_AES_on_an_FPGA.ipynb
+  """
+
+  # Key and plaintext generator
+  ktp = cw.ktp.Basic()
+  ktp.key_len = capture_cfg['key_len_bytes']
+  ktp.text_len = capture_cfg['plain_text_len_bytes']
+  ot.target.output_len = capture_cfg['plain_text_len_bytes']
+
+  key, text = ktp.next()
+
+  cipher = AES.new(bytes(key), AES.MODE_ECB)
+  print(f'Using key: {binascii.b2a_hex(bytes(key))}')
```

Comment:
Minor: If a hex string is acceptable, consider using `hex()` method of a `bytes` object, i.e. `bytes(key).hex()`. Applies to other usages as well.